### DISCUSSION TITLE
Clippy linted exercise

### DISCUSSION THREAD

**Discussion by keys-i** [rust-lang/rustlings#2143] [Q&A]
Could we perhaps make the exercises with clippy integrated into the whole thing?

**Comment by mo8it** [comment]
What do you mean with "integrated into the whole thing"?

**Comment by keys-i** [comment]
Many exercises include a variety of helpful clippy suggestions that could be incorporated to enhance them to be closer to idiomatic rust

**Comment by mo8it** [comment]
We already have Clippy exercises and Rustlings v6 shows Clippy lints to all other exercises as well.
